src/pages/blog/automate-where-it-matters-with-process-maps.pug

```diff
@@ -0,0 +1,143 @@
+            })
+            br
+
+            p From this rudimentary example, you can easily see:
+
+            ul
+              li The website gets 16,000 visitors a day from different sources.
+              li Most of those visitors drop out after landing on the website.
+              li Those which don't either phone, log in, or register.
+              li More drop out after this, only 2,500 a day place an order.
+              li Many orders suffer from payment failure, delivery failure, or a customer issue.
+              li About 1,500 customers receive successful deliveries per day.
+
+            p The beauty here is that the numbers don't need to be completely accurate in order for the map to be useful. We can work effectively with good guesses because we're analysing trends at a high level, not sweating the detail when we don't have it.
+
+            p However, should we get the detail, it could be added in the form of more nodes to get a more granular view of the business. For example, we may wish to dig into:
+
+            ul
+              li Where the leads who drop out from the website go.
+              li What happens to those who ring the call centre but never make a payment.
+              li What are the reasons for payments failing, and what those customers do next.
+              li What human activities happen between payment being accepted and the shoes being sent for delivery.
+
+            p These could be added by just building upon what's already included in the process map, without making the map any harder to understand for stakeholders who don't need such detailed information.
+
+            h3 Driving automation
+
+            p Process maps are fantastic tools for businesses looking to drive performance, efficiency, and scaleability through automation. They allow us to quickly see where inefficiencies lie, and allow us to make good predictions about what would happen if we were change things.
+
+            p For example, looking at our example e-commerce organisation again, conventional wisdom might tell us that in order to increase purchases, we need to increase marketing spend on advertising to drive more leads to the site.
+
+            p From the process diagram, however, we can see that lead numbers are not the whole picture. Half of the customers visiting the website drop out right away, and a further third end up on the phones, where they have a low chance to purchase, and we need humans available to take incoming calls.
+
+            p Following this logic, if we could keep the customers happy online, our process diagram tells us they'd be more likely to purchase, and would do so via a much cheaper channel.
+
+            p In conclusion, instead of simply increasing leads, we'd be much better off looking at why so many customers drop out from the website or resort to using the phone. We could then work to make it easier and more convenient for customers to carry on online.
```

Comment:
"In conclusion" makes me feel 'dissertation' rather than something I want to consume regularly.